src/mono/mono/mini/simd-arm64.h

```diff
@@ -0,0 +1,66 @@
+/* Remarks:
+ * - Mono encodes some vector operations with their dedicated opcodes (OP_XZERO) and others as subcodes within
+ *   umbrella opcodes (OP_NEGATION under OP_XOP_X_X). Use SIMD_OP and SIMD_OP_EX, respectively to encode those.
+ * - Operand config specifies the order of operands that are to be supplied to the function or macro. These are
+ *   variations of: W (width of the vector register), T (element type), D (dest reg number), S (source reg number),
+ *   I (immediate value). If _REV is specifed, the order of source registers is reversed. Note that not all 
+ *   options are supported. To specify more options, add the respective macros to the files that include this
+ *   (e.g. mini-arm64.c).
+ * - To specify that a particular operation is not supported for some data type, use  _UNDEF.
+ */
+
+/* 64-bit vectors */
+/*        Width   Opcode          Function              Operand config      I8                I16               I32               I64               F32               F64         */
```

Comment:
For max/min, there are different different intrinsics for signed vs unsigned integer types. For max, they are `arm_neon_smax` and `arm_neon_umax`. We need to either expend this table with more columns or add new macros to cover those cases. Which one do you think is the better option?

Reply:
I think we should have separate `inst_c0` for signed and unsigned int under `OP_XBINOP`. Int compare is also done like this. Then we would have a row for each of `OP_FMAX`, `OP_IMAX`, `OP_UMAX`.

Reply:
That's doable too.